server/application/application.proto

```diff
@@ -27,6 +34,11 @@ message ManifestQuery {
 
 message ApplicationResponse {}
 
+// ApplicationResourceEventsResponse holds a list of resource events.
+message ApplicationResourceEventsResponse {
+	repeated k8s.io.api.events.v1beta1.Event data = 1;
```

Comment:
Can we name the field `items` and rename this to EventsList? I checked and we are starting to become inconsistent with our response objects in the case of lists. We currently already have ApplicationList, ClusterList, and RepositoryList. I think we should continue that convention:

```
// EventsList holds a list of resource events.
message EventsList {
	repeated k8s.io.api.events.v1beta1.Event items = 1;
}
```

Similarly the repository.ListDirResponse should probably change as well. For consistency sake I think we should change from:

```
// ListDirResponse returns the contents of the repo of a ListDir request
message ListDirResponse {
    repeated string data = 1;
}
```
to:

```
// FileList returns the file names of the repo of a ListDir request
message FileList {
    repeated string items = 1;
}
```